SPECS/iptables/iptables.spec

```diff
@@ -96,41 +100,64 @@ rm -rf %{buildroot}/*
 %{_mandir}/man3/*
 
 %changelog
-* Sat May 09 00:21:34 PST 2020 Nick Samson <nisamson@microsoft.com> - 1.8.3-3
-- Added %%license line automatically
+* Wed Oct 21 2020 Rachel Menge <rachelmenge@microsoft.com> - 1.8.3-5
+- Modified the ip4, ip6 to match rules of other distros
+
+*   Thu Oct 15 2020 Rachel Menge <rachelmenge@microsoft.com> - 1.8.3-4
```

Comment:
Not a big deal, but the changes in this spec file could be in a single package revision, as the `-4` version wasn't released.  (The revision isn't really important as long as it's higher than the previous revision if the upstream version didn't change.)